charts/datasaur/values.yaml

```diff
@@ -353,4 +354,7 @@ elasticsearch:
   kibana:
     enabled: false
   storage: 100Gi
-  nodeCount: 3
\ No newline at end of file
+  nodeCount: 3
+
+clusterSettings:
```

Comment:
could you elaborate more about the irsa / pod identity here as comment?

Reply:
Have u tested if the IRSA becomes instance profile like ec2? @michael-jong-ds

Reply:
how do I check that ya niel?
in env there is this service account env from irsa
![Screenshot from 2024-03-06 10-44-17](https://github.com/datasaur-ai/helm-charts/assets/100671309/c58440fb-b660-4be5-bb2d-a75b0b7f2d1f)

Reply:
oh bisa pake aws sts get-called-identity aja mike

Reply:
ohh iya kali sts get-caller-identity sudah pakai arn service account irsa nya
```
{
    "UserId": "AROA6ODU75TFBCDY4G4CN:botocore-session-1709715567",
    "Account": "992382807242",
    "Arn": "arn:aws:sts::992382807242:assumed-role/eksIRSARole/botocore-session-1709715567"
}

```

Reply:
oh bisa pake aws sts get-called-identity aja mike